awscli/customizations/ec2addcount.py

```diff
@@ -64,9 +65,15 @@ def documentation(self):
     def add_to_parser(self, parser):
         parser.add_argument(self.cli_name, metavar=self.py_name,
                             help='Number of instances to launch',
-                            default='1')
+                            ## We will delegate the default value logic to
```

Comment:
Can this comment be moved out of the argument list and instead placed above the line?